geode-modules/src/main/java/org/apache/geode/services/module/impl/JBossModuleService.java

```diff
@@ -92,18 +98,20 @@ public boolean loadModule(ModuleDescriptor moduleDescriptor) {
         builder.addResourceRoot(ResourceLoaderSpec.createResourceLoaderSpec(resourceLoader));
       }
     } catch (IOException e) {
-      logger.error(e);
+      logger.error(e.getMessage());
       return false;
     }
 
+    builder.addDependency(DependencySpec.createSystemDependencySpec(PathUtils.getPathSet(null)));
+
     ModuleSpec moduleSpec = builder.create();
     moduleLoader.addModuleSpec(moduleSpec);
 
     try {
       modules.put(moduleDescriptor.getVersionedName(),
           moduleLoader.loadModule(moduleSpec.getName()));
     } catch (ModuleLoadException e) {
-      logger.error(e);
+      logger.error(e.getMessage());
```

Comment:
Please print the stacktrace. so please revert back to `logger.error(e)`

Reply:
Updated.